lib/Thruk/Authentication/User.pm

```diff
@@ -222,8 +222,22 @@ sub get_dynamic_roles {
     }
 
     if(!defined $can_submit_commands) {
-        $can_submit_commands = $c->config->{'can_submit_commands'} || 0;
-    }
+	if (defined $c->config->{'can_submit_commands'}) {
```

Comment:
this hole if block does exactly the same as before, just with more code.

Reply:
indeed...potentially a leftover during debugging where I placed some debug statements to trace further down